site/docs/patterns/international-phone-number-input.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: International Phone Number Input
+layout: DetailPattern
+sidebar:
+  label: International Phone Number Input
+aliases:
+  - /salt/patterns/international-phone-number-input
+data:
+  resources:
+    [
+      {
+        href: "https://www.figma.com/file/HkXrrf1rbYATx6ggGG4Pov/(WIP)-International-Phone-Number?node-id=1356%3A3430&mode=design",
+        label: "Templates",
+        internal: true,
+      },
+    ]
+---
+
+International Phone Number Input is a pattern that gives users a set of form fields to input phone number information for specific countries.
+
+## Pattern scope
+
+The pattern is a form that displays a relevant phone number field for a user-selected location. You can use it to provide consistent design when implementing International Phone Number Input interactions across your applications.
+
+We provide design guidance on how you can use Salt form control components to achieve the pattern. Your team can take further steps that fall outside its scope. For example, check with your tech partners if it’s possible to auto-detect user location, using data that can support auto-generating number placeholders, and save the entered number when the user changes the country.
+
+## When to use it
+
+Use the pattern when your user needs to enter a phone number for a specific country. This could be for contact or access request purposes, such as client detail collection processes or two-step verification/multi-factor authentication.
+
+## Pattern content
+
+- The pattern consists of two form controls: a [dropdown](../components/dropdown) that displays country selection (with a country symbol component thae user can expand or collapse), and an [input](../components/input) that displays the phone number.
+- When the user selects the country, the text input adornment in the phone number input will display the country code.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/international-phone-number-input/pattern_without_country.png",
+      alt: "Phone number input pattern without selected country",
+    },
+    {
+      src: "/img/patterns/international-phone-number-input/pattern_with_country.png",
+      alt: "Phone number input pattern with selected country",
+    },
+  ]}
+  label="Select United Kingdom"
+/>
+
+## Stack layout: column vs. row
+
+You can display the pattern in [stack layout](../components/stack-layout) as a column or a row.
```

Comment:
```suggestion
You can display the pattern using a [stack layout](../components/stack-layout) as a column or a row.
```

@sion-docs How should using components be referenced? It feels like this reads better but I'm not sure what the rules are here

Reply:
There is no relevant rule as yet, but I agree your suggestion reads better.